src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -138,10 +153,31 @@ const getWebhookSecret = async (uuid?: string): Promise<{ webhookSecret: string,
 			throw new Error(`Installation not found for gitHubApp with uuid ${uuid}`);
 		}
 		const webhookSecret = await gitHubServerApp.getDecryptedWebhookSecret(installation.jiraHost);
-		return { webhookSecret, gitHubServerApp };
+		/**
+		 * If we ever need to rotate the webhook secrets for Enterprise Customers,
+		 * we can add it in the array: ` [ webhookSecret ]`
+		 */
+		return { webhookSecrets: [ webhookSecret ], gitHubServerApp };
 	}
-	if (!envVars.WEBHOOK_SECRET) {
-		throw new Error("Environment variable 'WEBHOOK_SECRET' not defined");
+	if (allowGhCloudWebhookSecrets) {
+		if (!envVars.WEBHOOK_SECRETS) {
```

Comment:
Dont' need this, as we can put that in the env.ts check, fail-fast, make the app crash during startup.

Reply:
Removed from ehre